### DISCUSSION TITLE
I don't like the new UI without chat buttons. How do I undo or change a setting from like false to true etc. To revert?

### DISCUSSION THREAD

**Discussion by viperwasp** [oobabooga/text-generation-webui#4006] [Q&A] [answered]
In the settings settings or settings-template can I change or add a line to get the old style buttons back? I don't like the hover over menu?

**Comment by missionfloyd** [comment, marked as answer]
Go to the Session tab and enable `chat_buttons`, or use the `--chat-buttons` argument.

**Reply by viperwasp** [reply]
Thanks it worked.
